commit/merkleroot/types.go

```diff
@@ -16,11 +16,12 @@ type Query struct {
 }
 
 type Observation struct {
-	MerkleRoots        []cciptypes.MerkleRootChain     `json:"merkleRoots"`
-	OnRampMaxSeqNums   []plugintypes.SeqNumChain       `json:"onRampMaxSeqNums"`
-	OffRampNextSeqNums []plugintypes.SeqNumChain       `json:"offRampNextSeqNums"`
-	RMNRemoteConfig    rmntypes.RemoteConfig           `json:"rmnRemoteConfig"`
-	FChain             map[cciptypes.ChainSelector]int `json:"fChain"`
+	MerkleRoots        []cciptypes.MerkleRootChain      `json:"merkleRoots"`
+	RMNEnabledChains   map[cciptypes.ChainSelector]bool `json:"rmnEnabledChains"`
```

Comment:
Is this meant to be just a set or will this have all source chains w/ literally mapped true and false values?

RMNHome will not explicitly state that a source chain is not supported, that source chain would just be omitted.

Maybe would be clearer if this is `map[chainSel]struct{}` to signify that